gradient/cli/common.py

```diff
@@ -62,28 +62,35 @@ def get_option_name(options_strings):
         if opt.startswith("--"):
             return opt[2:]
 
-
 class ReadValueFromConfigFile(click.Parameter):
     def handle_parse_result(self, ctx, opts, args):
         config_file = ctx.params.get(OPTIONS_FILE_PARAMETER_NAME)
         if config_file:
             with open(config_file) as f:
                 config_data = yaml.load(f, Loader=yaml.FullLoader)
                 option_name = get_option_name(self.opts)
-                value = config_data.get(option_name)
-                if value is not None:
-                    if isinstance(value, dict):
-                        value = json.dumps(value)
-
-                    opts[self.name] = value
+                if (isinstance(self, GradientObjectListOption) and
+                    self.get_object_list_name() in config_data):
+                    value_list = []
+                    for object_list_item in config_data[self.get_object_list_name()]:
+                        value_list.append(object_list_item.get(self.get_object_key(), None))
+                    if len(value_list) > 0:
```

Comment:
if len(value_list) > 0:
does the same as:
if values_list:  # evaluates to True if iterable is not empty